swagger.yaml

```diff
@@ -5461,6 +5465,44 @@ definitions:
         type: string
         description: Ignition fetched from the specified API VIP
 
+  tang_connectivity_request:
+    type: object
+    required:
+      - tang_servers
+    properties:
+      tang_servers:
+        type: string
+        description: JSON-formatted string containing additional information regarding tang's configuration
+
+  tang_connectivity_response:
+    type: object
+    properties:
+      is_success:
+        type: boolean
+        description: Tang check result.
+      tang_server_response:
```

Comment:
are we using the tang_server_response somewhere? iiuc, we only check whether tang_connectivity_response succeeded for now?

Reply:
I use it in the agent code to make sure we get a valid response from Tang servers: see [TangRequest()](https://github.com/openshift/assisted-installer-agent/pull/341/files#diff-d1bab7abd87cd22a53866c28c672fb7386f432d9c44353543f3e3a4021afd987R74)

Reply:
oh ok, so TangServerSignatures is also mandatory for the response to be considered valid?

Reply:
Yes, this is an application-level validation (so it is not enough for the server to respond with 200 OK), with the specific thumbprint configured by the user.

WIth the agent code, we will Unmarshal the response we get from Tang, to make sure we get keys (Array) that are specific to the used thumbprint. This is to make sure as much as we can that upon reboot, things would work.